examples/react-app/src/index.tsx

```diff
@@ -100,6 +101,20 @@ ReactDOM.render(
           <h1>PlmgCard slot-2</h1>
         </div>
       </PlmgCard>
+      <br />
+      <PlmgRadioButtonGroup
```

Comment:
Wrap it into a Form with a submit button, so we can actually test and see the "required" behavior

Reply:
Awesome! Thank you very much :muscle: